dirs/dirs.go

```diff
@@ -126,9 +128,11 @@ func SetRootDir(rootdir string) {
 
 	switch release.ReleaseInfo.ID {
 	case "fedora", "centos", "rhel", "arch":
-		SnapMountDir = filepath.Join(rootdir, "/var/lib/snapd/snap")
+		BaseSnapMountDir = "/var/lib/snapd/snap"
```

Comment:
We can't use the "BaseSnap" term here, even more when we have a matching variable with "CoreSnap". Base snaps are around the block and this is unrelated.